mobile/lib/pages/editing/crop.page.dart

```diff
@@ -0,0 +1,185 @@
+import 'package:flutter/material.dart';
+import 'package:crop_image/crop_image.dart';
+import 'edit.page.dart';
+
+class CropImagePage extends StatefulWidget {
+  final Image image;
+  const CropImagePage({super.key, required this.image});
+  @override
+  State<CropImagePage> createState() => _CropImagePageState();
+
+}
+class _CropImagePageState extends State<CropImagePage> {
+  late CropController _cropController;
+  double? _selectedAspectRatio;
+  @override
+  void initState() {
+    super.initState();
+    _cropController = CropController(
+      defaultCrop: const Rect.fromLTRB(0.1, 0.1, 0.9, 0.9),
+    );
+  }
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.black,
+        leading: const CloseButton(color: Colors.white),
+        actions: [
+          IconButton(
+            icon: const Icon(Icons.done_rounded, color: Colors.white, size: 24),
+            onPressed: () async {
+              Image croppedImage = await _cropController.croppedImage();;
```

Comment:
`final croppedImage = `

Do we need to introduce some loading mechanism here? Or will `_cropController.croppedImage()` be very quick?

Reply:
It is pretty quick doesnot require a loading screen.